docs/helm.md

```diff
@@ -8,15 +8,52 @@ Tilt supports Helm out-of-the-box.
 
 There are two ways you can use Helm charts with Tilt.
 
-## Basic Helm
+- Install a remote Helm chart from the rich library of existing charts.
 
-If you have a local Helm chart and want to deploy it as-is, add a `helm()` call to your `Tiltfile`:
+- Iterate on a local Helm chart that you're building.
 
-```python
-k8s_yaml(helm('path/to/chart/dir'))
+Let's dig into each one.
+
+## Remote charts
+
+The `helm_remote` extension lets you load from a library of existing charts.
+
+First, add `.helm` to the `.tiltignore` file next to your `Tiltfile`. The
+`helm_remote` extension uses this directory as a cache, so you want to
+make sure that Tilt doesn't reload when it detects changes to this directory:
+
+```
+# .tiltignore
+
+.helm
+```
+
+Then call the `helm_remote` extension from your `Tiltfile`. Here's an example
+that deploys the `stable/mysql` chart:
+
+```
+# Tiltfile
+
+load('ext://helm_remote', 'helm_remote')
+helm_remote('mysql',
+            repo_name='stable',
+            repo_url='https://charts.helm.sh/stable')
 ```
 
-The `helm` function runs `helm template` on a chart directory and returns the generated yaml.
+For more options, see the [`helm_remote`
+README](https://github.com/tilt-dev/tilt-extensions/tree/master/helm_remote).
+They mostly mimic the options for using local charts.
```

Comment:
This line feels weird when the reader presumably has little context on the options for local charts at this point. Maybe just cut, or link to the section on local chart options.

Reply:
done